pkg/controller/controlplane/valuesprovider_test.go

```diff
@@ -435,6 +435,45 @@ var _ = Describe("ValuesProvider", func() {
 				azure.RemedyControllerName: remedyDisabled,
 			}))
 		})
+		It("should return correct control plane chart values when forcing read cache for in-tree PVs", func() {
+			shootAnnotations := map[string]string{
+				azure.ShootDiskConvertRWCachingModeAnnotation: "true",
+			}
+			cluster = generateCluster(cidr, k8sVersion, false, shootAnnotations, nil, &gardencorev1beta1.Seed{})
+
+			cp := generateControlPlane(controlPlaneConfig, infrastructureStatus)
+			values, err := vp.GetControlPlaneChartValues(ctx, cp, cluster, fakeSecretsManager, checksums, false)
+
+			Expect(err).NotTo(HaveOccurred())
+			Expect(values).To(Equal(map[string]interface{}{
```

Comment:
Maybe focus only on the values of interest so that the test won't break in case of changes on other fields.